docs/jupiterone.md

```diff
@@ -92,19 +97,19 @@ https://github.com/JupiterOne/sdk/blob/master/docs/integrations/development.md
 
 The following entities are created:
 
-| Resources | Entity `_type` | Entity `_class` |
-| --------- | -------------- | --------------- |
-| Account   | `acme_account` | `Account`       |
+| Resources       | Entity `_type`    | Entity `_class` |
+| --------------- | ----------------- | --------------- |
+| HubSpot Company | `hubspot_company` | `Organization`  |
+| HubSpot Role    | `hubspot_role`    | `AccessRole`    |
+| HubSpot User    | `hubspot_user`    | `User`          |
```

Comment:
I think we'll want to introduce a `hubspot_account` (**Account**) entity to encompass these entities. This will help us provide a more complete graph that doesn't have any floating entities.

Something like this:

```
hubspot_account HAS hubspot_company
hubspot_account HAS hubspot_role
hubspot_account HAS hubspot_user
```

> What data does my HubSpot organization contain?

```
FIND hubspot_account
THAT RELATES TO *
RETURN TREE
```